notebooks/contrib-dev/Fraction_of_Molecular_Collisions.ipynb

```diff
@@ -1062,10 +902,326 @@
+        "*hint: cast N[i] as an int*\n",
+        "\n",
+        "*hint: set plot = False*"
+      ],
+      "metadata": {
+        "id": "8YyCm6seysSw"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "points = 50\n",
+        "N = np.linspace(1, 50, points)\n",
+        "compare = np.zeros(points)\n",
+        "\n",
+        "### BEGIN SOLUTION\n",
+        "for i in range(points):\n",
+        "  i = int(i)\n",
+        "  compare[i] = midpoint_integration(int(N[i]), False)\n",
+        "  compare[i] -= result\n",
+        "### END SOLUTION\n",
+        "\n",
+        "print(compare)"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "p-tjM_UgyzOQ",
+        "outputId": "a89b29f8-60c4-4350-83d1-1957f26f5377"
+      },
+      "execution_count": 24,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "[-0.79312238 -0.21494489 -0.00351488  0.05427425  0.06668412  0.06553709\n",
+            "  0.06051562  0.05475037  0.0492623   0.04434694  0.04004378  0.03630544\n",
+            "  0.03306072  0.03023791  0.02777264  0.02560982  0.02370324  0.02201446\n",
+            "  0.02051161  0.01916824  0.01796231  0.0168754   0.01589206  0.01499924\n",
+            "  0.01418588  0.01344258  0.01276129  0.01213509  0.011558    0.01102485\n",
+            "  0.01053115  0.01007296  0.00964683  0.00924973  0.00887898  0.00853222\n",
+            "  0.00820733  0.00790246  0.00761591  0.0073462   0.00709197  0.00685202\n",
+            "  0.00662525  0.00641067  0.0062074   0.0060146   0.00583156  0.00565759\n",
+            "  0.00549207  0.00533445]\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### 6e. Plotting\n",
+        "Plot compare versus N. Make a publication quality figure.\n"
+      ],
+      "metadata": {
+        "id": "jOihDdotyjXG"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# plot the data\n",
+        "plt.figure(figsize=(6.4,4), dpi=100)\n",
+        "plt.grid()\n",
+        "\n",
+        "# write line that plots compare vs N\n",
+        "### BEGIN SOLUTION\n",
+        "plt.plot(N, compare, linewidth=3, label=\"Difference Between Gauss \\nQuadrature and Midpoint Rule\")\n",
+        "### END SOLUTION\n",
+        "plt.xticks(fontsize=15) #Tick font size 15\n",
+        "plt.yticks(fontsize=15)\n",
+        "plt.tick_params(direction=\"in\",top=True, right=True) #Major tick direction: in\n",
+        "plt.tick_params(which=\"minor\",direction=\"in\",top=True, right=True) #minor tick direction: in\n",
+        "plt.xlabel('Number of Midpoint Rule Rectangles',fontsize=16, fontweight = 'bold')\n",
+        "plt.ylabel('Difference Between Midpoint \\nRule and Gauss Quadrature \\nIntegral Results',fontsize=16, fontweight = 'bold')\n",
```

Comment:
Change the y-axis label to be shorter, such as "f_{mid}(x) - f_{quad)(x)". You also do not need a legend on this plot because the information is redundant with the title and x-axis. For the title, try "Comparison of Midpoint and Quadrature Rules"